.travis.yml

```diff
@@ -1,15 +1,76 @@
 language: python
 sudo: false
-python:
-    - 2.7
-    - 3.4
-    - 3.5
-    - 3.6
-    - pypy
-    - pypy3
+matrix:
+    include:
+        - os: linux
+          python: 2.7
+        - os: linux
+          python: 3.4
+        - os: linux
+          python: 3.5
+        - os: linux
+          python: 3.6
+        - os: linux
+          python: pypy
+        - os: linux
+          python: pypy3
```

Comment:
TBH I think you can leave the old python: ... list; the matrix: include: bits will be added to it.

Look at [zope.proxy](https://github.com/zopefoundation/zope.proxy/blob/master/.travis.yml) for an example (or zope.component, or zope.container).

Reply:
Okay, let's try it.